src/microbe_stage/Endosome.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Godot;
+using Newtonsoft.Json;
+
+/// <summary>
+///   This does nothing (for now) and only exist so saving could work.
+/// </summary>
+[JSONAlwaysDynamicType]
+[SceneLoadedClass("res://src/microbe_stage/Endosome.tscn", UsesEarlyResolve = false)]
+public class Endosome : Spatial, IEntity
+{
+    [JsonProperty]
+    private Color tint;
+
+    [JsonProperty]
+    private int renderPriority;
+
+    [JsonIgnore]
+    public MeshInstance? Mesh { get; private set; }
+
+    [JsonIgnore]
+    public Color Tint
+    {
+        get => tint;
+        set
+        {
+            tint = value;
+            ApplyTint();
+        }
+    }
+
+    [JsonIgnore]
+    public int RenderPriority
+    {
+        get => renderPriority;
+        set
+        {
+            renderPriority = value;
+            ApplyRenderPriority();
+        }
+    }
+
+    [JsonIgnore]
+    public Spatial EntityNode => this;
+
+    [JsonIgnore]
+    public AliveMarker AliveMarker { get; } = new();
+
+    public override void _Ready()
+    {
+        Mesh = GetNode<MeshInstance>("Vacuole");
+
+        // This has to be done here because setting this in Godot editor truncates
+        // the number to only 3 decimal places.
+        var material = Mesh?.MaterialOverride as ShaderMaterial;
```

Comment:
```suggestion
        Mesh = GetNode<MeshInstance>("Vacuole") ?? throw new Exception("mesh node not found");

        // This has to be done here because setting this in Godot editor truncates
        // the number to only 3 decimal places.
        var material = Mesh!.MaterialOverride as ShaderMaterial;
```

Btw why is the "Vacuole" node name used here?

Reply:
To differentiate it from the root which already has the name phagosome. Phagosome is technically a vacuole.

Reply:
I suspected that, but it took me like 30 seconds to realize that. I would name it something like "EngulfedObjectHolder" or something like that which is immediately obvious when reading the word and can't be confused by it being named after an organelle.

Reply:
> To differentiate it from the root which already has the name phagosome. Phagosome is technically a vacuole.

I actually meant Endosome not phagosome. Anyway changed it to your suggestion.